### DISCUSSION TITLE
Release 0.9.x

### DISCUSSION THREAD

**Discussion by EdgyEdgemond** [EdgyEdgemond/changelog-gen#84] [General]
@myddrin PR's all build on each other, so best to review (and merge) from bottom up :)

I think I'm done with changes for now. Have implemented all the things I have always felt were missing, as well as new things I've thought up over the last couple years.

If you want to take a look at the different PR's when you've got some time.

**Comment by EdgyEdgemond** [comment]
@LachlanMarnham not sure if I can ping you, 0.9 will add support for conventional commits and deprecate a bunch of configuration (as well as python 3.8)

**Comment by myddrin** [comment]
Will do, I started a bit on Friday, but you had a busy weekend I see!

**Comment by EdgyEdgemond** [comment]
All features that were missing have now been developed.

All previous features should still work, but upgrades to conventional commits and bump-my-version (active dev bumpversion fork) can be opted into.
